### PR TITLE
[P3][TERRAFORM] Setup d'un cloud front pour le bucket static

### PR DESCRIPTION
**Pull request by jocelyn-q**
image
image

### CODE REVIEW

File: terraform/P3/6-create-cloud-front.tf [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  default_root_object = "index.html"
+
+  default_cache_behavior {
+    allowed_methods        = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
+    cached_methods         = ["GET", "HEAD", "OPTIONS"]
+    target_origin_id       = aws_s3_bucket.tf_front_end_bucket.bucket
+    viewer_protocol_policy = "redirect-to-https"
+    compress               = true
+
+    min_ttl     = 0
```
Comment (EmericRENAUD) [inline terraform/P3/6-create-cloud-front.tf:16]: ttl = time to live ?

File: terraform/P3/6-create-cloud-front.tf [hunk 2]
```diff
@@ -0,0 +1,53 @@
+resource "aws_cloudfront_origin_access_identity" "tf_cloudfront_oai" {
+  comment = "tf_cloudfront"
```
Comment (EmericRENAUD) [inline terraform/P3/6-create-cloud-front.tf:2]: ça sert à quoi ça ?

**Review by EmericRENAUD — APPROVED** [review]

File: terraform/P3/6-create-cloud-front.tf (hunk 2 above)
Comment (jocelyn-q) [reply terraform/P3/6-create-cloud-front.tf:2]: Amazon CloudFront Origin Access Identity (OAI) is a virtual user identity used for sharing private content via CloudFront. It is used to give the CF distribution permission to fetch a private object from the origin server, such as an S3 bucket. To work with Origin Access Identities, navigate to Clouds > AWS Global > CF Origin Access Identities. The CloudFront Origin Access Identities page lists all OAIs that were created by the RightScale account

Comment (jocelyn-q) [reply terraform/P3/6-create-cloud-front.tf:2]: En gros c'est un identity check

**Pull request merged**